addons/website/static/src/js/editor/wysiwyg_multizone.js

```diff
@@ -37,7 +37,7 @@ HelperPlugin.include({
     isOnNonHTMLField: function () {
```

Comment:
This method was supposed to check if the field is not html so we can be sure never to drop html in a field that doesn't accept it and drop text instead. This would make that you can drop html and images into char and monetary fields, which I don't think is the goal. I have trouble understanding the bug you're trying to fix though. Could you please make a video of it so I can assist you better? Thank you.